### PR TITLE
Integrate rapidjson into ci

### PR DESCRIPTION
**Pull request by ccaraman**

### CODE REVIEW

File: ci/build_container/build_container.sh [hunk 1]
```diff
@@ -131,4 +131,7 @@ wget -O gcovr-3.3.tar.gz https://github.com/gcovr/gcovr/archive/3.3.tar.gz
 tar xf gcovr-3.3.tar.gz
 rm gcovr-3.3.tar.gz
 
-
+# rapidjson
```
Comment (mattklein123) [inline ci/build_container/build_container.sh]: move before test dependencies

**Comment by ccaraman** [comment]
@lyft/network-team

File: thirdparty.cmake [hunk 2]
```diff
@@ -50,6 +50,10 @@ set(ENVOY_LIGHTSTEP_TRACER_INCLUDE_DIR "" CACHE FILEPATH "location of lighstep t
 # Extra linker flags required to properly link envoy with all of the above libraries.
 set(ENVOY_EXE_EXTRA_LINKER_FLAGS "" CACHE STRING "envoy extra linker flags")
 
+# https://github.com/miloyip/rapidjson
```
Comment (RomanDzhabarov) [inline thirdparty.cmake]: nit: can you group code dependency in one place? (just place this before extra linker flags)

**Comment by RomanDzhabarov** [comment]
lgtm, one small comment

File: thirdparty.cmake (hunk 2 above)
Comment (ccaraman) [inline thirdparty.cmake]: Moved it up.

**Pull request merged**